### DISCUSSION TITLE
CSS help

### DISCUSSION THREAD

**Discussion by echoriiku** [FreshRSS/FreshRSS#5932] [Q&A]
How can I change the font of the title and content? I want to use the Custom CSS extension to do that but I don't have much coding knowledge :(

If it's possible, title and content with different fonts.

![image](https://github.com/FreshRSS/FreshRSS/assets/42903366/58d8c69f-a1f6-4a43-94d0-dca97ba40b86)

**Comment by Frenzie** [comment]
If you right click, inspect, you can see the CSS styles that currently apply.

![image](https://github.com/FreshRSS/FreshRSS/assets/202757/9d61eea3-2774-479f-9782-a97418c1d067)

After that you can copy the selector(s) and add `font-family: "Some Font"`.

**Comment by sherlcok314159** [comment]
When I want to add external font, how should I import the font?

**Reply by math-GH** [reply]
let's discuss your question in your own thread here: https://github.com/FreshRSS/FreshRSS/discussions/6194
